src/js/edu-benefits/5490/config/form.js

```diff
@@ -72,9 +73,7 @@ const {
   ssn
 } = fullSchema5490.definitions;
 
-const nonRequiredFullName = _.assign(fullName, {
```

Comment:
The logic here has been updated to provide an empty object as the initial argument, so as not to overwrite the `fullName` schema.